easybuild/easyconfigs/c/CDAT/CDAT-8.2.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,132 @@
+easyblock = 'PythonBundle'
+
+name = 'CDAT'
+version = '8.2.1'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'https://github.com/CDAT/cdat'
+description = """CDAT is a powerful and complete front-end to a rich set of 
+visual-data exploration and analysis capabilities well suited for data analysis problems."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('Qt5', '5.14.1'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.4'),
+    ('libxml2', '2.9.10'),
+    ('libxslt', '1.1.34'),
+    ('SQLite', '3.31.1'),
+    ('HDF5', '1.10.6'),
+    ('bzip2', '1.0.8'),
+    ('OpenSSL', '1.1.1e'),
```

Comment:
@zarybnicky You shouldn't include `OpenSSL` as a dependency provided through EasyBuild, that should be provided by the OS, so security updates are picked up...

You should include it as an OS dependency instead:

```python
osdependencies = [OS_PKG_OPENSSL_DEV]

Reply:
I'm pretty sure those are just leftover dependencies from when I took over the EC, I'll try to prune them